package.json

```diff
@@ -12,8 +12,8 @@
   "scripts": {
     "test": "mocha -r ts-node/register tests/**/*.spec.ts",
     "lint": "resin-lint --typescript lib examples tests",
-    "build": "tsc",
-    "install": "tsc"
+    "build": "rm -rf build && tsc",
```

Comment:
This will break on Windows (no `rm` there) – I'd suggest to maybe use [`rimraf`](https://github.com/isaacs/rimraf), and make that `"build": "rimraf build && tsc"`

Reply:
Good catch, updated.